node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
Okay, I will proceed with it. May I know what were your thoughts against loc2 approach. I know that adding such functions for purpose of compat is not good. But given that we are dealing with a function which takes variable number of arguments, it seemed okay to me. thoughts?

I might not take it up in next couple of days and I am stuck with a high priority work. Hope that is fine.